### PR TITLE
Portfolio setup

### PR DESCRIPTION
**Pull request by zairdon20**
# Project Setup

### In the following milestone, I have:

- Made sure that there are no linter errors:

- Made sure that linters are set up correctly for this repository.

- Made sure that linter results are green.

- Made sure there are no custom changes to linter config.

- Made sure that I used the correct GitHub Flow:

- Made sure the README file is descriptive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,28 +4,22 @@
 
 > One paragraph statement about the project.
 
-![screenshot](./app_screenshot.png)
+![screenshot](img\127.0.0.1_5500_index.html.png)
 
 Additional description about the project and its features.
```
Comment (Gambit142) [inline README.md]: - [ ] I would like to suggest that you fix the broken image for the screenshot of this project. This is the way it looks presently. Kindly fix this

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h2>welcome</h2>
```
Comment (Gambit142) [inline index.html]: - [ ] Nice job implementing this, Kindly put the _**welcome text**_ in an `a` tag. The welcome text serves as the logo for the project. Ideally, the logo of any project should be clickable and should link to the homepage of a website. You can also make the `W` in the `welcome` to be in uppercase Kindly fix this. 🙏

File: style.css [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+.section-1 {
+  background-image: url(img/header-illsutration-mobile.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center;
+  width: 100%;
+  height: 100vh;
+}
```
Comment (Gambit142) [inline style.css]: - [ ] One of the project requirements is that you use flexbox to style the header and section in this portfolio. Kindly implement this specific project requirement for the section tag in your headline.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+            don’t hestiate to
+                </p>
+        <ul class="icons">
+            <li><i class="fa fa-twitter" aria-hidden="true"></i></li>
+            <li><i class="fa fa-linkedin" aria-hidden="true"></i></li>
+            <li><i class="fa fa-medium" aria-hidden="true"></i></li>
+            <li><i class="fa fa-github" aria-hidden="true"></i></li>
+            <li><i class="fa fa-angellist" aria-hidden="true"></i></li>
+        </ul>
+    </section>
```
Comment (Gambit142) [inline index.html:34]: - [ ] I would like to suggest that you work on the background images for the section part. One of the project requirements states that it should be the same as that of Figma's template. 

### Your portfolio
image

### Figma's Template
image

Kindly work on this.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  margin-top: 35px;
+}
+
+.icons {
+  list-style: none;
+  display: flex;
+  justify-content: space-around;
+  margin: 0 50px;
+  margin-top: 30px;
+}
```
Comment (Gambit142) [inline style.css]: - [ ] Kindly make the icons the same size as that of Figma's template. You could make the size of the icon either `20px` or `1.25rem`

**Review by Gambit142 — CHANGES_REQUESTED** [review]
## Hi @zairdon20 👋, 

Good job so far!

## To Highlight
- Good pull request description ✔️ 
- Good start setting up the README.md (although it can be improved)
- Correct use of Github flow ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by VanessaAoki — APPROVED** [review]
## STATUS: APPROVED 🟢 
Hi @zairdon20, 

Great job implementing all the changes required by the previous Code Reviewer, your project is complete! There is nothing else to say other than... it's time to merge it! 
Congratulations! 🎉 

approved

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Your README file looks very professional and the PR has a descriptive title/summary of the implemented changes :memo: 
- [x] Descriptive commits and correct use of the GitHub Flow :fire: 


Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**